libPhoneNumber/NBPhoneNumberOfflineGeocoder.m

```diff
@@ -0,0 +1,173 @@
+//
+//  NBPhoneNumberOfflineGeocoder.m
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 ohtalk.me. All rights reserved.
+//
+
+#import "NBPhoneNumberOfflineGeocoder.h"
+
+@implementation NBPhoneNumberOfflineGeocoder {
+ @private
+  NBPhoneNumberUtil *phoneUtil;
+  NBGeocoderMetadataHelper *geocoderHelper;
+}
+
+- (instancetype)init {
+  self = [super init];
+  self->phoneUtil = NBPhoneNumberUtil.sharedInstance;
+  // gather all available language database files
+  NSBundle *bundle = [NSBundle bundleForClass:self.classForCoder];
+  NSURL *bundleURL = [[bundle resourceURL] URLByAppendingPathComponent:@"Resources.bundle"];
+  NSArray *filesInResourceBundle =
+      [[NSFileManager defaultManager] contentsOfDirectoryAtURL:bundleURL
+                                    includingPropertiesForKeys:NULL
+                                                       options:0
+                                                         error:NULL];
+  NSMutableArray *supportedLanguages = [[NSMutableArray alloc] init];
+  [filesInResourceBundle enumerateObjectsUsingBlock:^(id obj, NSUInteger idx, BOOL *stop) {
+    NSString *filename = (NSString *)obj;
+    if (filename != NULL) {
```

Comment:
Objective-C objects uses `nil` instead of `NULL`

Reply:
done.